addons/colorblind/addon.json

```
@@ -83,16 +88,12 @@
           "name": "Dotted"
         }
       ],
-      "default": "solid",
-      "if": {
-        "settings": { "links": "underline" }
-      }
+      "default": "solid"
```

The default settings of "customizable XYZ" addons are usually the same as Scratch's defaults.
```suggestion
      "default": "none"
```